openscenario/openscenario_interpreter/src/syntax/position.cpp

```diff
@@ -43,6 +43,7 @@ Position::operator geometry_msgs::msg::Pose() const
       // clang-format off
       [&](const         WorldPosition & position) { return static_cast<geometry_msgs::msg::Pose>(position); },
       [&](const RelativeWorldPosition & position) { return static_cast<geometry_msgs::msg::Pose>(position); },
+      [&](const RelativeObjectPosition & position) { return static_cast<geometry_msgs::msg::Pose>(position); },
```

Comment:
Adjust the column alignment according to the style of the original code. Do not try to break the style to reduce diffs.

Reply:
fixed https://github.com/tier4/scenario_simulator_v2/pull/1030/commits/7824f61848f356476a46d9e9555c9b820df1d519